app/views/layouts/_admin_dropdown.html.erb

```diff
@@ -6,37 +6,27 @@
     </span>
   <% end %>
   <ul class="dropdown-menu">
-    <% if can? :manage, :all  %>
-      <li class="dropdown-submenu">
-        <%= link_to(t(:admin), nil) %>
-        <ul class="dropdown-menu">
-          <li><%= link_to(models_name(User), admin_users_path) %></li>
-          <li><%= link_to(models_name(Menu), admin_menus_path) %></li>
-          <li><%= link_to(models_name(Constant), constants_path) %></li>
-          <li><%= link_to(models_name(Permission), admin_permission_users_path) %></li>
-        </ul>
-      </li>
-    <% end %>
-    <% if can? :modify, Contact %>
-      <li class="dropdown-submenu">
-        <%= link_to models_name(Contact), contacts_path %>
-      </li>
-    <% end %>
-    <% if can? :modify, News %>
-      <li class="dropdown-submenu">
-        <%= link_to models_name(News), admin_news_index_path %>
-      </li>
-    <% end %>
+    <li class="dropdown-submenu">
+      <%= link_to(t(:admin), nil) %>
```

Comment:
Only those with admin permission `:manage, :all` can access this menu now, so we can (finally!) remove the submenu if we want to.

Doing so would make my life a little easier, since I would no longer have to guess the position of the submenues when trying to visit the admin pages from my phone :wink: